WordPress/Classes/Categories/NSString+Helpers.m

```diff
@@ -188,4 +188,13 @@ - (NSArray *)tokenize
     return tokens;
 }
 
+- (BOOL)isWordPressComURL
```

Comment:
Would it make sense to use the term "Path" instead of "URL" for consistency with other NSString path methods?